### PR TITLE
JCB-based obs+bias staging

### PR DESCRIPTION
**Pull request by DavidNew-NOAA**
This PR is a companion to Global Workflow PR [#2992](https://github.com/NOAA-EMC/global-workflow/pull/2992) and GDASApp PR [#1312](https://github.com/NOAA-EMC/GDASApp/pull/1312).

It add observation and bias files staging templates required for the above Global Workflow PR.

### CODE REVIEW

File: algorithm/atmosphere/atm_bias_staging.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,13 @@
+mkdir:
+    - '{{atmosphere_obsbiasout_path}}'
+copy:
+    {% for observation_from_jcb in observations %}
+    {% if use_observer(observation_from_jcb) %}
+    {% set ob %}
+    {% include observation_from_jcb + '.yaml.j2' %}
+    {% endset %}
+    {% if 'obs bias' in ob %}
+    - ['{{atmosphere_obsbiasroot_path}}/{{atmosphere_obsbiasin_prefix}}rad_varbc_params.tar', '{{atmosphere_obsbiasin_path}}']
```
Comment (CoryMartin-NOAA) [inline algorithm/atmosphere/atm_bias_staging.yaml.j2]: this only needs to be copied once, right? what if multiple observers have `obs bias`, will this copy it N times? Is there a way to 'exit' the loop?

Comment (DavidNew-NOAA) [reply algorithm/atmosphere/atm_bias_staging.yaml.j2]: @CoryMartin-NOAA Yeah, good question. There's no ```break``` statement in Jinja2. The way I worked around this was by removing the redundant entries from the bias_dict here in the GW companion PR: https://github.com/NOAA-EMC/global-workflow/blob/057a5e95a61109bf4c5fd583e3c0903cedace5b9/ush/python/pygfs/task/atm_analysis.py#L142

**Comment by CoryMartin-NOAA** [comment]
@DavidNew-NOAA @danholdaway this is a good opportunity to figure out why the JCB-GDAS CI times out after hours. It hangs and doesn't work properly and I don't know what is going on (we can force merge like I was doing in Dan's absence)

**Review by CoryMartin-NOAA — APPROVED** [review]

File: algorithm/atmosphere/atm_bias_staging.yaml.j2 (hunk 1 above)
Comment (DavidNew-NOAA) [reply algorithm/atmosphere/atm_bias_staging.yaml.j2]: @CoryMartin-NOAA Now I'm second guessing myself. The way I wrote the code was for the the possibility that different obs will have different bias tarballs in the future. Should I just assume that all biases will be in that one tarball?

**Comment by RussTreadon-NOAA** [comment]
@DavidNew-NOAA:  As currently scripted all radiance bias correction files are in a single tarball.  It sounds like aircraft bias correction will be a single file for all tail numbers.  Is this correct, @CoryMartin-NOAA ?

**Comment by CoryMartin-NOAA** [comment]
@RussTreadon-NOAA that is my understanding, yes

**Comment by DavidNew-NOAA** [comment]
@CoryMartin-NOAA @RussTreadon-NOAA If that's the case, should we put an entry in each the observation templates, under the ```obs_bias``` entry, something like ```tarball```, that indicates the name of the tarball, rather than hardcoding it into the JCB bias staging template? See here for example: https://github.com/NOAA-EMC/jcb-gdas/blob/2b46aa19b22df22f33ef0be5030fb719a8fa6298/observations/atmosphere/atms_npp.yaml.j2#L37

**Comment by CoryMartin-NOAA** [comment]
@DavidNew-NOAA that might interfere with the JEDI YAML readers?

**Comment by DavidNew-NOAA** [comment]
@CoryMartin-NOAA See my latest commit. Also https://github.com/NOAA-EMC/GDASApp/pull/1312/commits/29696c82ab994e02dcfbeefc4b30953de352605b in my GDASApp PR. I think this is going to be the best way to do this.

**Review by danholdaway — APPROVED** [review]

**Comment by danholdaway** [comment]
I can put it on my list for next week to fix the CI. I don't have a lot of time this week with the ongoing sprint and don't want to hold things up.

**Pull request merged**